### ISSUE TITLE
game bugs 1

### ISSUE THREAD

**Issue by DavidWhaleMEF** [Microsoft/pxt#1921]
microbit.org-ticket: 1286

Customer has reported the following bugs with the game module:

- if I create a sprite then I need two invocation of the delete() statement so to remove it from the screen -

**Comment by DavidWhaleMEF** [comment]
Customer writing us a repro .hex at moment

**Comment by abchatra** [comment]
Need repro steps.

**Comment by pelikhan** [comment]
repro
```
let item: game.LedSprite = null
item = game.createSprite(2, 2)
input.onButtonPressed(Button.A, () => {
    item.delete();
})
```

**Comment by pelikhan** [comment]
Fix in 0.12.53

**Comment by pelikhan** [comment]
Fix in 0.11.10
